static/app/types/detectors.tsx

```diff
@@ -0,0 +1,119 @@
+export type DetectorType = 'metric' | 'crons' | 'uptime';
```

Comment:
it might be nice to split this file up a bit more; instead of `types/detectors.tsx` having it be `types/workflowEngine/detectors.tsx` and then having a separate file for our 3 models (data source, detector, workflow)

Reply:
Refactored in f661a95d653